src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -139,15 +139,23 @@ HRESULT MicrosoftInstrumentationEngine::CProfilerManager::SetupProfilingEnvironm
     // to co create a free threaded version of msxml on a thread that it owns to avoid this.
     CHandle hConfigThread(CreateThread(NULL, 0, InstrumentationMethodThreadProc, this, 0, NULL));
 
-    DWORD retVal = WaitForSingleObject(hConfigThread, 15000);
-    if (retVal == WAIT_ABANDONED)
+    DWORD waitTime = 60 * 1000; // Wait 1 minute for loading instrumentation methods
+#ifdef Debug
```

Comment:
I'm wondering if we need to parameterize this at some point.

Reply:
+1, Definitely something to consider for the next iteration

---
In reply to: [308869827](https://github.com/microsoft/CLRInstrumentationEngine/pull/111#discussion_r308869827) [](ancestors = 308869827)